fdbserver/workloads/AuthzSecurity.actor.cpp

```diff
@@ -135,6 +154,11 @@ struct AuthzSecurityWorkload : TestWorkload {
 		m.push_back(publicNonTenantRequestPositive.getMetric());
 		m.push_back(tLogReadNegative.getMetric());
 		m.push_back(keyLocationLeakNegative.getMetric());
+		m.push_back(bgLocationLeakNegative.getMetric());
```

Comment:
Since we evaluate/update these metrics only with `checkBlobGranules == true`, the metric collection should also follow suit.